meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect --format='{{.Id}}' $SUPERVISOR_IMAGE)
+SUPERVISOR_CONTAINER_IMAGE_ID=$(docker inspect --format='{{.Image}}' resin_supervisor)
+
+runSupervisor() {
+    docker rm --force resin_supervisor || true
```

Comment:
Why do we try to start it afterwards if it cannot remove the old supervisor? Should we error out if we need to start a new supervisor but we can't remove the current container?